libs/search-widget/src/widgets/row/Row.svelte

```diff
@@ -163,16 +163,22 @@
     position: relative;
     width: 100%;
     height: 0;
-    padding-top: 50%;
+    padding-top: 80%;
+    background-color: var(--color-dark-light);
+  }
+  .thumbnail.semantic {
+    background-color: #f0f0f0;
```

Comment:
this is `color-neutral-lighter` in our palette https://nuclia.github.io/frontend/palette
I know we use svelte for our widget, but maybe we could find a way to have our CSS variables corresponding to the same palette in both our widget and apps.
It's ok to fix that in another PR though, as we'll probably have to apply such a fix to a lot of places in the widget

Reply:
Sounds good, it make sense to use the same colors.